v7/recent_posts_json/recent_posts_json.py

```diff
@@ -131,7 +132,10 @@ def make_json(self, posts, descriptions, previewimage, output_path, lang):
             recent_posts.append(entry)
         data = json.dumps(recent_posts, indent=2, sort_keys=True)
         with io.open(output_path, "w+", encoding="utf8") as outf:
-            outf.write(data)
+            if sys.version_info[0] != 3:
```

Comment:
This is not right.

On Python 2, json.dumps() might return Unicode in some cases. It doesn’t by default; yet we could use to change the default:

```py
data = json.dumps(recent_posts, ensure_ascii=False, indent=2, sort_keys=True)
with io.open(output_path, "w+", encoding="utf-8") as outf:
    try:
        outf.write(data.decode('utf-8'))
    except AttributeError:
        outf.write(data)
````